static/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-agent: *
+Disallow:
+Sitemap: https://modrinth.com/sitemap.xml
```

Comment:
Why specify a sitemap when one isn't added in this PR? As far as I can tell, https://knossos-que7rdys2-modrinth.vercel.app/sitemap.xml is still a 404